cmd/main.go

```diff
@@ -112,7 +113,9 @@ func main() {
 	)
 
 	foundProject, version := getProjectVersion()
-	if foundProject && version == "1" {
+	if foundProject && version == project.Version1 {
```

Comment:
```suggestion
	if hasProjectFile && projectVersion == project.Version1 {
```
WDYT?

Reply:
Why do you suggest to do this change from another PR (#1143) into this?

Reply:
Hi @Adirio, 

Because this PR is doing/replacing `if foundProject && version == "1" {` for `if foundProject && version == project.Version1 {`. 

So, as part of its review, I am suggesting `if foundProject && version == "1" {` for `if hasProjectFile && projectVersion == project.Version1 {` instead of.

Reply:
I agree that @camilamacedo86 suggested naming is more suitable. But I also agree that we can lay it on #1143, as the narrow scope of my change was only to replace the string `"1"` with the constant `project.Version1`, and not to change other things.

If you still think we should introduce this change here, let me know.

Reply:
@hypnoglow The change she is suggesting is not just a more suitable name, it has to do with the package with the same alias (`version`). The context is missing as this is a change I suggested in #1143 and she suggested a name change. Suggesting to change this here makes no sense as this PR is about a completely different topic.

Reply:
It is not a big deal so I am OK with.